platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/Cryptography.java

```diff
@@ -129,24 +129,6 @@ default Hash getNullHash() {
      */
     Hash getNullHash(final DigestType digestType);
 
-    /**
-     * Verifies the given digital signatures for authenticity. The result of the verification will be returned by the
-     * {@link TransactionSignature#getSignatureStatus()} method once the future (available via
-     * {@link TransactionSignature#getFuture()}) has been completed.
-     * <p>
-     * Note: This implementation is non-blocking and returns almost immediately.
-     * <p>
-     * Starting in version 0.43 and onwards, the {@link SignatureType#ECDSA_SECP256K1} signature algorithm requires the
-     * payload to be a KECCAK-256 hash of the original message. Verification will fail if the message is not 32 bytes in
-     * length and the output of 256-bit hashing function.
-     *
-     * @param signatures a list of signatures to be verified
-     * @deprecated use {@link #verifySync(List)} instead, asynchronous verification is a feature that is scheduled for
-     * removal
-     */
-    @Deprecated(forRemoval = true)
-    void verifyAsync(final List<TransactionSignature> signatures);
```

Comment:
When we remove `verifyAsync(...)` it would make make sense to rename `verifySync(...)` to `verify(...)`.

Reply:
I would be in favor of eventually removing the whole module. A lot of it is based on our legacy serialization which we are moving away from.

Reply:
@hendrikebbers 

> Code looks good. I ask myself if performance tests are needed since you moved from a non blocking api to a blocking api at several code places.

There are no production use cases of async verification, only testing apps and mono-services (which is no longer production code).